packages/kotti-ui/source/kotti-button/hooks/use-actions-tippy.ts

```diff
@@ -0,0 +1,73 @@
+import { useTippy } from '@3yourmind/vue-use-tippy'
+import { Instance } from 'tippy.js'
+import { Ref, computed, ref, watch } from 'vue'
+
+import { TIPPY_VERTICAL_OFFSET } from '../../constants'
+
+export const useActionsTippy = ({
+	isDisabled,
+	isLoading,
+}: {
+	isDisabled: Ref<boolean>
+	isLoading: Ref<boolean>
```

Comment:
It may be a good idea to watch on these and hide the tippy when the values change to true.